spi/core-spi/build.gradle.kts

```diff
@@ -30,6 +30,7 @@ dependencies {
 
     implementation("io.opentelemetry:opentelemetry-api:${openTelemetryVersion}")
 
+    api(project(":common:util"))
```

Comment:
unless you're exposing classes from `:common:util` in this module's public API, it's better to use `implementation` dependencies